.gitignore

```diff
@@ -1,2 +1,23 @@
+# Inteliji #
 .idea/
 target/
+out/
+
+*.iml
+*.iws
+
+# Package Files #
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+### Java ###
+# Compiled class file
+*.class
+
+# Log file
+*.log
```

Comment:
Do you also see that red sign over here? :) It requires you to add a new line at the end of the file :)

Reply:
This just sucks. Sorry for it. I enabled the conforming setting in Intelijii, it should help me get rid of these forever. At least, I hope so :)